build.gradle

```diff
@@ -443,8 +462,10 @@ afterEvaluate {
             binary.tasks.withType(LinkSharedLibrary) { builderTask ->
                 nativeJar.into("net/rubygrapefruit/platform/$variantName") { from builderTask.linkedFile }
             }
-            project.tasks.test {
-                classpath.from nativeJar
+            if (!testVersionFromLocalRepository) {
+                project.tasks.test {
+                    classpath.from nativeJar
+                }
             }
```

Comment:
Can we turn this into lazy configuration somehow?

Reply:
Ah, so you want to avoid realizing the test task? I think I can do that.

Reply:
Done.